pyproject.toml

```diff
@@ -113,10 +98,43 @@ deepspeech = [
     "hydra-core",
 ]
 
+math = [
+    "numpy",
+    "pandas",
+    "scipy >= 1.4.1",
+    "scikit-learn < 1.1.0",   # ART requires scikit-learn >=0.22.2,<1.1.0
+    "matplotlib",
+]
+
+datasets = [
+    "armory-testbed[tensorflow]",
+    "tensorflow-datasets >= 4.6.0",
+    "protobuf",
+]
+
+datasets-builder = [
+    "armory-testbed[datasets,engine]",
+    "apache-beam >= 2.22.0",
+]
+
```

Comment:
Is there a reason these all got reordered? Hard to follow changes.

Reply:
This change moves the frameworks to the top so that it is easier to find their dependencies.